app/models/data_file.rb

```diff
@@ -180,6 +182,15 @@ def related_website_list
     self.related_websites.collect(&:url).join("|")
   end
 
+  def validate_related_websites
+    related_websites = related_website_list.split("|")
```

Comment:
The existing code split related websites on '|' so I follow that. Should I refactor code for related websites?